### PR TITLE
Add inline loading component

### PR DESCRIPTION
**Pull request by msecret**
Decided to add it as a loading component modifier. Had to add some
extra inline rules to the documentation because the main loading
component is absolute and takes over all the space. Also added
all images to docs, as it's required for documenting these components.

TODO: change the background of the styleguide.

As seen here: https://github.com/18F/cg-design/issues/14

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "build-js": "mkdir -p ./js && browserify ./src/js/main.js -o ./js/cloudgov-style.js",
     "build-prefix": "postcss --use autoprefixer css/cloudgov-style.css -o ./css/cloudgov-style.css",
     "build-img": "mkdir -p ./img && npm run copy-img && npm run svg",
-    "build-library-assets": "npm run clean && npm run build-css && npm run compile-css && cp ./css/cloudgov-style-min.css ./docs/assets/css && npm run copy-font && cp ./fonts/* ./docs/assets/fonts",
+    "build-library-assets": "npm run clean && npm run build-css && npm run compile-css && cp ./css/cloudgov-style-min.css ./docs/assets/css && npm run copy-font && cp ./fonts/* ./docs/assets/fonts && npm run build-img && npm run build-img && cp -R ./img/* ./docs/assets/img",
```
Comment (jcscottiii) [inline package.json]: you have `npm run build-img` twice.

**Comment by jcscottiii** [comment]
just one thing. otherwise it's good.

whew. next time let's break up into separate PRs. was a little tough to scroll through all the given stuff vs what I needed to look at. but this feedback is also coming from a person who also creates mega-patches lol.

**Review by jcscottiii — CHANGES_REQUESTED** [review]

**Comment by jcscottiii** [comment]
feel free to merge after fixing that one thing.

**Comment by adborden** [comment]
:+1: For the future, it would be easier to review if you kept all the assets and autogenerated files to a separate commit, so it's easier to filter out the core of the changes you're making.

**Pull request merged**